model/compile/QtiAssetReplacer/NullQtiItemAssetReplacer.php

```
@@ -33,7 +33,11 @@ public function shouldBeReplaced(PackedAsset $packetAsset): bool
         return false;
     }
 
-    public function replace(PackedAsset $packetAsset, $itemId): PackedAsset
+    public function replace(
+        PackedAsset $packetAsset,
+        string $itemId,
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$itemId'. (undefined)

(UnusedFormalParameter)